### DISCUSSION TITLE
is prisma seed deprecated?

### DISCUSSION THREAD

**Discussion by probablyVincentIp** [prisma/prisma#4687] [Q&A] [answered]
I was _attending_ the presentation by @tomhoule about `prisma migrate --preview-feature` where he was using a seed file to   populate a demo database, when I wanted to look into this I could only find [prisma v1 documentation](https://v1.prisma.io/docs/1.34/prisma-cli-and-configuration/cli-command-reference/prisma-seed-xcv8/), with no mention in the newer documentation?

**Comment by tomhoule** [comment, marked as answer]
Hi @supermooshman — the seeding in my presentation was just done with an npm script in `package.json`. something like `"seed": "ts-node seed.ts"`, which you can run with `npm run seed`. `seed.ts` is a regular typescript file, where I imported the prisma client. It is not a built-in prisma feature. If you have specific ideas about seeding, feedback issues are very welcome :)

**Reply by probablyVincentIp** [reply]
Thanks fo the reply Tom.

Just to pick your brain for another second, how do you handle the use of foreign keys with this method? Do you make sure that these are created first in your script, or is there more/less to it?

**Reply by tomhoule** [reply]
Since it was only for a demo, it was done in a very naive way: create the parent records, save the ids in variables, then connect the child records when they are created.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
